testing/unit/test_faucetizer.py

```diff
@@ -319,6 +319,123 @@ def test_devices_learned_and_authenticated(self):
         self._verify_behavioral_config(expected_config)
 
 
+class FaucetizerBehaviorWithoutTailACLTestCase(FaucetizerTestBase):
+    """Test Faucetizer's behavior with no tail_acl setting in Forch config"""
+
+    ORCH_CONFIG = """
+    unauthenticated_vlan: 100
+    """
+
+    FAUCET_STRUCTURAL_CONFIG = """
```

Comment:
this seems very basic (the structural config) -- should it be promoted to a superclass to prevent repetition?  Is there anything unique to this test case about it?